roachprod/install: TestServiceDescriptors failed

roachprod/install.TestServiceDescriptors [failed](https://mesolite.cluster.engflow.com/invocations/default/6b86bb21-f33b-4bc6-8ef5-3f85a74faa0f?testReportRun=10&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvcm9hY2hwcm9kL2luc3RhbGw6aW5zdGFsbF90ZXN0) on release-25.3.5-rc @ [adc4b2ff64df47187d363c76db80a5ed4200aa35](https://github.com/cockroachdb/cockroach/commits/adc4b2ff64df47187d363c76db80a5ed4200aa35):


```
=== RUN   TestServiceDescriptors
    services_test.go:430: 
        	Error Trace:	pkg/roachprod/install/services_test.go:430
        	            				GOROOT/src/reflect/value.go:584
        	            				GOROOT/src/reflect/value.go:368
        	            				GOROOT/src/testing/quick/quick.go:290
        	            				pkg/roachprod/install/services_test.go:468
        	Error:      	Not equal: 
        	            	expected: 26257
        	            	actual  : 29000
        	Test:       	TestServiceDescriptors
--- FAIL: TestServiceDescriptors (0.40s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=10</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/test-eng
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestServiceDescriptors.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-56991

--- a/pkg/roachprod/install/services_test.go
+++ b/pkg/roachprod/install/services_test.go
@@ -138,7 +138,7 @@ func (t *serviceRegistryTest) makeVM(clusterName string, i int) vm.VM {
 }
 
 func (t *serviceRegistryTest) newCluster(nodeCount int) *SyncedCluster {
-	clusterName := fmt.Sprintf("cluster-%d", t.rng.Uint32())
+	clusterName := fmt.Sprintf("cluster-%s", randutil.RandString(t.rng, 10, randutil.PrintableKeyAlphabet))
 	c := &SyncedCluster{
 		Cluster: cloud.Cluster{
 			Name: clusterName,

